blivet/devices/disk.py

```diff
@@ -332,6 +351,17 @@ def add_parent(self, parent):
         else:
             self.parents.append(parent)
 
+    def _add_parent(self, parent):
+        super()._add_parent(parent)
+        if Tags.remote not in self.tags and Tags.remote in parent.tags:
+            self.tags.add(Tags.remote)
+
+    def _remove_parent(self, parent):
+        super()._remove_parent(parent)
+        if Tags.remote in self.tags and Tags.remote in parent.tags and \
+           len(p for p in self.parents if Tags.remote in p.tags) < 2:
```

Comment:
``not any(p for p in self.parents if Tags.remote in p.tags and p != parent)`` would be better readable for me here, but that might be just a matter of personal preference.